docs/new-integration-how-to-integrate-azure-administrative-units.md

```diff
@@ -0,0 +1,77 @@
+---
+id: new-integration-how-to-integrate-azure-administrative-units
+title: How to Integrate Azure with Administrative Units
+---
+
+:::note What is this guide about? This guide shows you how to set up an advanced
+Azure platform integration using Administrative Units. This approach limits
+meshStack's permissions to a specific scope within your Entra tenant and is recommended to improve the security of your integration.:::
+
+## What are Administrative Units?
+
+[Administrative Units](https://docs.microsoft.com/en-us/azure/active-directory/roles/administrative-units)
+are containers in Microsoft Entra ID (formerly Azure AD) that allow you to
+delegate administrative permissions to a restricted scope of resources. Instead
+of granting organization-wide permissions to meshStack, you can limit access to
+specific users, and groups within an Administrative Unit that meshStack is
+allowed to manage.
+
+## Benefits of Using Administrative Units with meshStack
+
+Here's a comparison of the permissions required for meshStack's replicator service principal
+when using Administrative Units versus the standard integration:
+
+<!--
+TODO:
```

Comment:
According to these docs we need this: https://learn.microsoft.com/en-us/graph/api/administrativeunit-post-members?view=graph-rest-1.0&tabs=http#permissions-to-create-a-new-group